docs/source/user_guide/options.rst

```diff
@@ -240,7 +240,10 @@ compute.max_rows                1000           'compute.max_rows' sets the limit
                                                When the limit is set, it is executed by the shortcut
```

Comment:
I think it's easier to just document that `'compute.max_rows' sets the limit of the current DataFrame.` -> `compute.max_rows' sets the limit of the current Koalas DataFrame.`.

It is documented that `groupby.apply` passes pandas DataFrame so explicitly mentioning it is a Koalas DataFrame will clarify all similar cases.

Reply:
If I understand correctly, even though `compute.max_rows` rows will be loaded into memory, Koalas will still ensure that if a group exceeds `compute.max_rows` rows, it will pass a Pandas DataFrame with all rows the the function(s). I think emphasizing this is an important, too. Any suggestions?